skflow/estimators/stacked_autoencoder.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import tensorflow as tf
+from skflow.io.data_feeder import get_batch
+
+class StackedAutoEncoder:
```

Comment:
how about using `fit(x)` for fitting encoder and `transform(x)` for encoding data? and `fit_transform` for fitting and encoding (similar to your encode). Also is there a way to use TensorFlowBaseEstimator as base class?